GillianCore/logging/report.ml

```diff
@@ -1,49 +1,63 @@
+type uuidm = Uuidm.t
+
+let yojson_of_uuidm uuidm = Uuidm.to_string uuidm |> yojson_of_string
+
+let uuidm_of_yojson yojson =
+  string_of_yojson yojson |> Uuidm.of_string |> Option.get
+
 module PackedPP : sig
-  type t
+  type t [@@deriving yojson]
```

Comment:
The deriving yojson should probably go away ?

Reply:
If we remove it, then `[@@deriving yojson]` for `type agnostic_content` will complain because it cannot find `PackedPP.yojson_of_t` and `PackedPP.t_of_yojson` in the signature of `PackedPP`. I guess those could be added in manually as well.

Reply:
![image](https://user-images.githubusercontent.com/8451179/82659611-df738d80-9c20-11ea-92ce-764c2ff7896c.png)
My bad